frontend/packages/core/src/NPS/feedback.tsx

```diff
@@ -158,6 +158,7 @@ const NPSFeedback = (opts: FeedbackOptions) => {
           origin: IClutch.feedback.v1.Origin[opts.origin],
           userSubmitted: hasSubmit,
           survey,
+          ...(window.location.search && { urlSearchParams: window.location.search }),
```

Comment:
super nit: this seems like additional complexity for little benefit. what is the benefit over just doing
```suggestion
          urlSearchParams: window.location.search,
```

Reply:
i had used this approach for a scenario where the object key being present mattered but yeah in this case, no real benefit. will update!